demo/shaders/glsl/sprite.glsl

```diff
@@ -9,9 +9,9 @@ layout(set = 0, binding = 0) uniform Args {
 //             mag_filter: Linear,
 //             min_filter: Linear,
 //             mip_map_mode: Linear,
-//             address_mode_u: Repeat,
-//             address_mode_v: Repeat,
-//             address_mode_w: Repeat,
+//             address_mode_u: Mirror,
```

Comment:
Repeat was causing the color of the edge of one side of the texture to show up on the other side